go-controller/pkg/ovn/egressip.go

```diff
@@ -820,6 +836,26 @@ func findReroutePolicyIDs(filterOption, egressIPName string, gatewayRouterIPs []
 	return strings.Split(policyIDs, "\n"), nil
 }
 
+func findLegacyReroutePolicyIDs() ([]string, error) {
+	policyIDs, stderr, err := util.RunOVNNbctl(
+		"--format=csv",
+		"--data=bare",
+		"--no-heading",
+		"--columns=_uuid",
+		"find",
+		"logical_router_policy",
+		fmt.Sprintf("priority=%v", types.EgressIPReroutePriority),
+		"nexthop!=[]",
+	)
+	if err != nil {
+		return nil, fmt.Errorf("unable to find legacy logical router policies, stderr: %s, err: %v", stderr, err)
+	}
+	if policyIDs == "" {
+		return nil, nil
+	}
+	return strings.Split(policyIDs, "\n"), nil
```

Comment:
even though format is csv, it will be separated by new line? I think this is right, but just want to double check with you

Reply:
Yes, a csv and table format will return the same result in this case:

```
# ovn-nbctl --format csv --no-heading --columns _uuid find logical_router_policy
f39fc950-996e-493a-9e7c-5dd561f5384e
224ee4cb-220d-4bc9-8203-fd554e58b885
66b7d2cb-66b3-462b-8087-70f3e1f1965f
696f80d4-6fb5-434d-9b34-effd1e403032
43d4f8fc-1b3b-4e9c-bd34-95c0a0ad929b
ca5f3dcc-1e08-4035-beff-d1186ba889f6
3894fb15-f36a-4c28-8a38-b29d81d0d280
163e1415-39aa-4fa4-a963-721d6d2bf973
9acc7d1f-dae9-44f3-8590-05599dab3d84
[root@ovn-control-plane ~]# ovn-nbctl --format table --no-heading --columns _uuid find logical_router_policy
f39fc950-996e-493a-9e7c-5dd561f5384e
224ee4cb-220d-4bc9-8203-fd554e58b885
66b7d2cb-66b3-462b-8087-70f3e1f1965f
696f80d4-6fb5-434d-9b34-effd1e403032
43d4f8fc-1b3b-4e9c-bd34-95c0a0ad929b
ca5f3dcc-1e08-4035-beff-d1186ba889f6
3894fb15-f36a-4c28-8a38-b29d81d0d280
163e1415-39aa-4fa4-a963-721d6d2bf973
9acc7d1f-dae9-44f3-8590-05599dab3d84
```